README.md

```diff
@@ -4,22 +4,22 @@
 
 ---
  
-Eve is a programming language and IDE based on years of research into building a human-first programming platform. You can play with Eve online here: [play.witheve.com](http://play.witheve.com/).
+Eve is a programming language based on years of research into building a human-first programming platform. 
 
-[![Play With Eve](http://witheve.github.io/assets/images/editor.png)](http://play.witheve.com/#/examples/flappy.eve)
+**This repository hosts a preview of Eve v0.3 alpha, which is currently under active development. You can use Eve on its own or integrate it into an exixsting project, but be aware this preview contains bugs and lacks documentation. For help with this preview release, please contact the devlopers on the Eve [mailing list](https://groups.google.com/forum/#!forum/eve-talk)**
 
-## How to use Eve
+## Getting Started with Eve v0.3 preview
 
 Install [Node](https://nodejs.org/en/download/) for your platform, then install Eve:
 
 ```
-npm install -g witheve
+npm install -g witheve@preview
```

Comment:
Let's just direct people to pull the starter in wholesale for now, with
```sh
git clone git@github.com:witheve/eve-starter.git
cd eve-starter
npm install
```

They can then access the included examples with the program switcher:
```sh
npm start
```

Or run a specific program from anywhere with:
```sh
npm start -- path/to/program.js
```

Once they're comfortable with how Eve works, they can either modify the starter to suit (for building new programs) or integrate Eve into their existing program (using your instructions, but dropping the `-g` flag on the install).